[BUG] panics with --numPduSessions=16

When PacketRusher is started with numPduSessions set to 16, the process crashes when it attempts to establish the 16th PDU session.

## To Reproduce

Extract the archive: [1.tgz](https://github.com/user-attachments/files/19475011/1.tgz)

```bash
docker compose build
docker compose up -d
docker logs -f gnb0
# wait approximately 40 seconds until PacketRusher crashes
```

## Actual behavior
PacketRusher completes registration and establishes 15 PDU sessions, and then crashes with error:

```text
panic: runtime error: index out of range [16] with length 16

goroutine 78 [running]:
my5G-RANTester/internal/control_test_engine/gnb/context.(*GNBUe).GetSelectedNssai(...)
	/PacketRusher/internal/control_test_engine/gnb/context/ue.go:150
my5G-RANTester/internal/control_test_engine/gnb/ngap.HandlerPduSessionResourceSetupRequest(0xc0002561e0, 0x4?)
	/PacketRusher/internal/control_test_engine/gnb/ngap/handler.go:388 +0x120e
my5G-RANTester/internal/control_test_engine/gnb/ngap.Dispatch(0xc0002480f0, 0xc0002561e0, {0xc0003226c0, 0xb2, 0xb2})
	/PacketRusher/internal/control_test_engine/gnb/ngap/dispatcher.go:51 +0x5cd
created by my5G-RANTester/internal/control_test_engine/gnb/ngap.GnbListen in goroutine 6
	/PacketRusher/internal/control_test_engine/gnb/ngap/service.go:85 +0x6d
```

Note: after the crash, before you can test again, it's necessary to unload and re-load the gtp5g kernel module:
```bash
sudo sh -c 'rmmod gtp5g; modprobe gtp5g'
```

## Expected behavior
PacketRusher establishes the 16th PDU session successfully.

## Architecture
- OS: Ubuntu 24.04
- Platform: amd64
- Version: PacketRusher 542ddc649b5d3fcac85e8f2ee58d45d5fe07fa17
- Kernel: 6.8.0-55-lowlatency
- Docker: 28.0.2
- gtp5g: v0.9.12

## Logs

PacketRusher logs:
```text
time="2025-03-26T20:19:49Z" level=info msg="Selecting 172.25.198.16 for host 172.25.198.16 as AMF's IP address"
time="2025-03-26T20:19:49Z" level=info msg="Selecting 172.25.195.18 for host 172.25.195.18 as gNodeB's N3/Data IP address"
time="2025-03-26T20:19:49Z" level=info msg="Selecting 172.25.198.18 for host 172.25.198.18 as gNodeB's N2/Control IP address"
time="2025-03-26T20:19:49Z" level=info msg="Loaded config at: /config.gnb0.001017005551000.yml"
time="2025-03-26T20:19:49Z" level=info msg="PacketRusher version 1.0.1"
time="2025-03-26T20:19:49Z" level=info msg=---------------------------------------
time="2025-03-26T20:19:49Z" level=info msg="[TESTER] Starting test function: Testing registration of multiple UEs"
time="2025-03-26T20:19:49Z" level=info msg="[TESTER][UE] Number of UEs: 1"
time="2025-03-26T20:19:49Z" level=info msg="[TESTER][GNB] gNodeB control interface IP/Port: 172.25.198.18/9487~"
time="2025-03-26T20:19:49Z" level=info msg="[TESTER][GNB] gNodeB data interface IP/Port: 172.25.195.18/2152"
time="2025-03-26T20:19:49Z" level=info msg="[TESTER][AMF] AMF IP/Port: 172.25.198.16/38412"
time="2025-03-26T20:19:49Z" level=info msg=---------------------------------------
time="2025-03-26T20:19:49Z" level=info msg="[GNB] SCTP/NGAP service is running"
time="2025-03-26T20:19:49Z" level=info msg="[GNB] Initiating NG Setup Request"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][NGAP] Receive NG Setup Response"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][AMF] AMF Name: amf"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][AMF] State of AMF: Active"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][AMF] Capacity of AMF: 255"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][AMF] PLMNs Identities Supported by AMF -- mcc: 001 mnc:01"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][AMF] List of AMF slices Supported by AMF -- sst:01 sd:000000"
time="2025-03-26T20:19:49Z" level=info msg="[GNB][AMF] List of AMF slices Supported by AMF -- sst:04 sd:000000"
time="2025-03-26T20:19:50Z" level=info msg="[TESTER] TESTING REGISTRATION USING IMSI 7005551000 UE"
time="2025-03-26T20:19:50Z" level=info msg="[GNB] Received incoming connection from new UE"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating Registration"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Switched from state 0 to state 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive Downlink NAS Transport"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message without security header"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Receive Authentication Request"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS][MAC] Authenticity of the authentication request message: OK"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS][SQN] SQN of the authentication request message: VALID"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Send authentication response"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Switched from state 1 to state 2"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive Downlink NAS Transport"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with security header"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with integrity and with NEW 5G NAS SECURITY CONTEXT"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] successful NAS MAC verification"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Receive Security Mode Command"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Type of ciphering algorithm is 5G-EA0"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Type of integrity protection algorithm is 128-5G-IA2"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive Initial Context Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][UE] UE Context was created with successful"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][UE] UE RAN ID 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][UE] UE AMF ID 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][UE] UE Mobility Restrict --Plmn-- Mcc: 001 Mnc: 01"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][UE] UE Masked Imeisv: 1110000000ffff00"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][UE] Allowed Nssai-- Sst: [01] Sd: [000000]"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][AMF] Send Initial Context Setup Response."
time="2025-03-26T20:19:50Z" level=info msg="[GNB] Initiating Initial Context Setup Response"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] No PDU Session to set up in InitialContextSetupResponse."
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with security header"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with integrity and ciphered"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] successful NAS CIPHERING"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] successful NAS MAC verification"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Receive Registration Accept"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] UE 5G GUTI: &{119 11 [242 0 241 16 1 0 0 0 0 0 1]}"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Switched from state 2 to state 3"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating New PDU Session"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive Downlink NAS Transport"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with security header"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with integrity and ciphered"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] successful NAS CIPHERING"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] successful NAS MAC verification"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Receive Configuration Update Command"
time="2025-03-26T20:19:50Z" level=info msg="[UE] Initiating Configuration Update Complete"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 2"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB] Initiating PDU Session Resource Setup Response"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with security header"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Message with integrity and ciphered"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] successful NAS CIPHERING"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] successful NAS MAC verification"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Receive DL NAS Transport"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] Receiving PDU Session Establishment Accept"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] PDU session QoS RULES: [1 0 6 49 49 1 1 255 1]"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] PDU session DNN: internet"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] PDU session NSSAI -- sst: 1 sd: 000"
time="2025-03-26T20:19:50Z" level=info msg="[UE][NAS] PDU address received: 10.1.0.1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 2"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 2"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 3"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 6"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 3"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 6"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 12"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 5"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 7"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 14"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 6"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 5"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: 01 sd: 000000"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 10"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 7"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 16"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 10"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 20"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 10"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 18"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 11"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: 01 sd: 000000"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 22"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 11"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 12"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 24"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 11"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 13"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] NSSAI Selected --- sst: NSSAI was not selected sd: NSSAI was not selected"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Type: ipv4"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] QOS Flow Identifier: 1"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Uplink Teid: 26"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Downlink Teid: 13"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Non-Dynamic-5QI: 9"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] Priority Level ARP: 8"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] UPF Address: 172.25.195.5 :2152"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][SCTP] Receive message in 0 stream\n"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP] Receive PDU Session Resource Setup Request"
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session was created with successful."
time="2025-03-26T20:19:50Z" level=info msg="[GNB][NGAP][UE] PDU Session Id: 16"
panic: runtime error: index out of range [16] with length 16

goroutine 78 [running]:
my5G-RANTester/internal/control_test_engine/gnb/context.(*GNBUe).GetSelectedNssai(...)
	/PacketRusher/internal/control_test_engine/gnb/context/ue.go:150
my5G-RANTester/internal/control_test_engine/gnb/ngap.HandlerPduSessionResourceSetupRequest(0xc0002561e0, 0x4?)
	/PacketRusher/internal/control_test_engine/gnb/ngap/handler.go:388 +0x120e
my5G-RANTester/internal/control_test_engine/gnb/ngap.Dispatch(0xc0002480f0, 0xc0002561e0, {0xc0003226c0, 0xb2, 0xb2})
	/PacketRusher/internal/control_test_engine/gnb/ngap/dispatcher.go:51 +0x5cd
created by my5G-RANTester/internal/control_test_engine/gnb/ngap.GnbListen in goroutine 6
	/PacketRusher/internal/control_test_engine/gnb/ngap/service.go:85 +0x6d
```

## Additional context

PacketRusher config:
```yaml
amfif:
  - ip: 172.25.198.16
    port: 38412
gnodeb:
  controlif:
    ip: 172.25.198.18
    port: 9487
  dataif:
    ip: 172.25.195.18
    port: 2152
  plmnlist:
    gnbid: "000001"
    mcc: "001"
    mnc: "01"
    tac: "000005"
  slicesupportlist:
    sd: "000000"
    sst: "01"
logs:
  level: 4
ue:
  amf: "8000"
  ciphering:
    nea0: true
    nea1: false
    nea2: true
    nea3: false
  dnn: internet
  homeNetworkPublicKey: 5a8d38864820197c3394b92613b20b91633cbd897119273bf8e4a6f4eec0a650
  homeNetworkPublicKeyID: 1
  hplmn:
    mcc: "001"
    mnc: "01"
  integrity:
    nia0: false
    nia1: false
    nia2: true
    nia3: false
  key: ea9c7340dea02fcfcec55f250f2fa714
  msin: "7005551000"
  opc: 0f958f6ec042db70e7d5d44b1b6460dd
  protectionScheme: 0
  routingindicator: "0000"
  snssai:
    sd: "000000"
    sst: 1
  sqn: "00000000"
```

PacketRusher invocation command line:
```bash
packetrusher --config=/config.gnb0.001017005551000.yml multi-ue -n=1 -d -t --tunnel-vrf=false --numPduSessions=16
```

--- a/internal/control_test_engine/gnb/context/ue.go
+++ b/internal/control_test_engine/gnb/context/ue.go
@@ -147,7 +147,7 @@ func (ue *GNBUe) GetUeMaskedImeiSv() string {
 }
 
 func (ue *GNBUe) GetSelectedNssai(pduSessionId int64) (string, string) {
-	pduSession := ue.context.pduSession[pduSessionId]
+	pduSession := ue.context.pduSession[pduSessionId-1]
 	if pduSession != nil {
 		return pduSession.sst, pduSession.sd
 	}

